### PR TITLE
Added console message implementation for ESP8266.

### PR DESCRIPTION
**Pull request by slaff**

### CODE REVIEW

**Comment by yichoi** [comment]
@slaff Travis failed, please include [DCO](https://github.com/Samsung/jerryscript/wiki/JerryScript-Developer's-Certificate-of-Origin)

**Comment by slaff** [comment]
@yichoi Should be better now. Thanks for noticing this.

File: targets/esp8266/user/jerry_port.c [hunk 1]
```diff
@@ -28,8 +29,7 @@ jerry_port_console (const char *format, /**< format string */
 {
   va_list args;
   va_start (args, format);
-  /* TODO, uncomment when vprint link is ok */
-  /* vfprintf (stdout, format, args); */
+  ets_vprintf(ets_putc, format, args);
```
Comment (LaszloLango) [inline targets/esp8266/user/jerry_port.c]: missing space before `(`

File: targets/esp8266/user/jerry_port.c [hunk 2]
```diff
@@ -18,6 +18,7 @@
 #include <stdarg.h>
 
 #include "jerry-core/jerry-port.h"
+extern int ets_putc(int);
```
Comment (LaszloLango) [inline targets/esp8266/user/jerry_port.c]: ditto

**Comment by zherczeg** [comment]
LGTM

**Comment by LaszloLango** [comment]
LGTM

**Pull request merged**